sorts/cocktail_shaker_sort.py

```diff
@@ -42,4 +42,4 @@ def cocktail_shaker_sort(unsorted: list) -> list:
     doctest.testmod()
     user_input = input("Enter numbers separated by a comma:\n").strip()
     unsorted = [int(item) for item in user_input.split(",")]
-    print(f"{cocktail_shaker_sort(unsorted) = }")
+    print(f"{cocktail_shaker_sort(unsorted) }")
```

Comment:
Why break this?

Reply:
I was getting error in pycharm 'unexpected expression part' before running a program.

Reply:
I have fixed it again and I have commited changes. Sorry for the inconvenience